files/build_templates/docker_image_ctl.j2

```diff
@@ -121,7 +128,7 @@ start() {
         fi
 
         # docker created with a different HWSKU, remove and recreate
```

Comment:
Suggest update comment also: `# docker created with a different HWSKU and/or hostname, remove and recreate`

Reply:
done